updater/spec/dependabot/dependency_change_builder_spec.rb

```diff
@@ -0,0 +1,121 @@
+                    ],
+                    experiments: {})
+  end
+
+  let(:dependency_files) do
+    [
+      Dependabot::DependencyFile.new(
+        name: "Gemfile",
+        content: fixture("bundler/original/Gemfile"),
+        directory: "/"
+      ),
+      Dependabot::DependencyFile.new(
+        name: "Gemfile.lock",
+        content: fixture("bundler/original/Gemfile.lock"),
+        directory: "/"
+      )
+    ]
+  end
+
+  let(:updated_dependencies) do
+    [
+      Dependabot::Dependency.new(
+        name: "dummy-pkg-b",
+        package_manager: "bundler",
+        version: "1.2.0",
+        previous_version: "1.1.0",
+        requirements: [
+          {
+            file: "Gemfile",
+            requirement: "~> 1.2.0",
+            groups: [],
+            source: nil
+          }
+        ],
+        previous_requirements: [
+          {
+            file: "Gemfile",
+            requirement: "~> 1.1.0",
+            groups: [],
+            source: nil
+          }
+        ]
+      )
+    ]
+  end
+
+  describe "::create_from" do
+    subject(:create_change) do
+      described_class.create_from(
+        job: job,
+        dependency_files: dependency_files,
+        updated_dependencies: updated_dependencies,
+        change_source: change_source
+      )
+    end
+
+    context "when the source is a lead dependency" do
+      let(:change_source) do
+        Dependabot::Dependency.new(
+          name: "dummy-pkg-b",
+          package_manager: "bundler",
+          version: "1.1.0",
+          requirements: [
+            {
+              file: "Gemfile",
+              requirement: "~> 1.1.0",
+              groups: [],
+              source: nil
+            }
+          ]
+        )
+      end
+
+      it "creates a new DependencyChange with the updated files" do
+        dependency_change = create_change
+
+        expect(dependency_change).to be_a(Dependabot::DependencyChange)
+        expect(dependency_change.dependencies).to eql(updated_dependencies)
+        expect(dependency_change.updated_dependency_files.map(&:name)).to eql(["Gemfile", "Gemfile.lock"])
+        expect(dependency_change).not_to be_grouped_update
+
+        gemfile = dependency_change.updated_dependency_files.find { |file| file.name == "Gemfile" }
+        expect(gemfile.content).to eql(fixture("bundler/updated/Gemfile"))
+
+        lockfile = dependency_change.updated_dependency_files.find { |file| file.name == "Gemfile.lock" }
+        expect(lockfile.content).to eql(fixture("bundler/updated/Gemfile.lock"))
+      end
+    end
+
+    context "when the source is a group rule" do
+      let(:change_source) do
+        Dependabot::GroupRule.new(name: "dummy-pkg-*")
+      end
+
+      it "creates a new DependencyChange flagged as a grouped update" do
+        dependency_change = create_change
+
+        expect(dependency_change).to be_a(Dependabot::DependencyChange)
+        expect(dependency_change).to be_grouped_update
```

Comment:
I know rspec will automatically generate some matchers like this. 
It doesn't look like `grouped_update` exists on the DependencyChange or DependencyChangeBuilder classes though, so where does it come from?